### DISCUSSION TITLE
tiles in start menu

### DISCUSSION THREAD

**Discussion by CorruptionVR** [DustinBrett/daedalOS#256] [Q&A] [answered]
is a tiled start menu on the todo list? what I mean by tiled is like the windows 10 tiles.

**Comment by DustinBrett** [comment, marked as answer]
It has indeed been on my to do list in the past. But to be honest I recently took a poll on what was top of mind for the community and myself and I've thrown out most of my to do list in the hopes of not overwhelming myself and focusing on the core things.

That being said I will consider your mention of it as a +2 for adding that to the list. Thanks for asking!
